test/integration/model.test.js

```diff
@@ -2561,3 +2561,30 @@ describe(Support.getTestDialectTeaser('Model'), function() {
     return this.sequelize.sync({force: true});
   });
 });
+
+  it('should return array of errors if validate and individualHooks are true in bulkCreate', function() {
```

Comment:
Few style issues in this test, we generally use two space indentation with leading spaces.